### DISCUSSION TITLE
Where can I find python version support ?

### DISCUSSION THREAD

**Discussion by Andromelus** [pytest-dev/pytest#8690] [Q&A] [answered]
I am stuck with python 3.5.2 (company policy).

I would like to use pytest, but when installing via pip, I get the following error:

`pytest requires Python '>=3.6' but the running Python is 3.5.2`

I would like to know where I can find the information telling me which version of pytest I should install along with python 3.5.2. I have not been able to find it on the official website (I hva to admit that I have not been reading all the changelogs...).

Is there a web page giving this information ?

**Comment by The-Compiler** [comment, marked as answer]
Did you try upgrading your pip? With a new enough pip, it should automatically install the latest available version. FWIW, support for 3.5 was dropped in pytest 6.2.0: https://docs.pytest.org/en/6.2.x/changelog.html#pytest-6-2-0-2020-12-12

Since Python 3.5 [has been end-of-life](https://devguide.python.org/devcycle/#end-of-life-branches) in September 2020 (including all security support!), consider advocating to changing that company policy, though...
